### PR TITLE
Trigger EMR on EKS jobs with Apache Airflow

### PR DESCRIPTION
**Pull request by season1946**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [x] Yes, I have added a new example under [examples](https://github.com/awslabs/data-on-eks/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR to add a new Add-on for [Terraform EKS Blueprints](https://github.com/aws-ia/terraform-aws-eks-blueprints) repo (if applicable)
- [x] Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by season1946** [comment]
@vara-bonthu  the pre-commit failed because of the external modules not this example code. Pls review and I am ready to demo it.

File: schedulers/managed-airflow-mwaa/eks.tf [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  # 3. IAM role for the team execution role
+  # 4. Update AWS_AUTH config map with  emr-containers user and AWSServiceRoleForAmazonEMRContainers role
+  # 5. Create a trust relationship between the job execution role and the identity of the EMR managed service account
+  #---------------------------------------
+  enable_emr_on_eks = true
+  emr_on_eks_teams = {
+    emr-wmaa-team = {
+      namespace               = "emr-wmaa"
+      job_execution_role      = "emr-eks-wmaa-team"
+      additional_iam_policies = [aws_iam_policy.emr_on_eks.arn]
```
Comment (vara-bonthu) [inline schedulers/managed-airflow-mwaa/eks.tf:84]: replace `wmaa` with `mwaa`. This needs changing in few other places as well

File: website/docs/job-schedulers-eks/aws-managed-airflow.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ sidebar_position: 1
 
 # Amazon Manged Workflows for Apache Airflow (MWAA)
```
Comment (vara-bonthu) [inline website/docs/job-schedulers-eks/aws-managed-airflow.md:6]: Let's add Introduction about MWAA with few sentences here. You can pick aws MWAA content

File: schedulers/managed-airflow-mwaa/eks.tf [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+# Kubernetes Add-on Module
+#------------------------------------------------------------------------
+module "eks_blueprints_kubernetes_addons" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.10.0"
+
+  eks_cluster_id       = module.eks_blueprints.eks_cluster_id
+  eks_cluster_endpoint = module.eks_blueprints.eks_cluster_endpoint
+  eks_oidc_provider    = module.eks_blueprints.oidc_provider
+  eks_cluster_version  = module.eks_blueprints.eks_cluster_version
+
```
Comment (vara-bonthu) [inline schedulers/managed-airflow-mwaa/eks.tf:101]: Lets add managed add-ons as well vpc cni, coredns, kube-proxy and csi driver

**Review by vara-bonthu — COMMENTED** [review]
Thanks for this great PR 👍🏼 @season1946 . Overall PR looks good to me. I left few minor comments.

**Comment by season1946** [comment]
@vara-bonthu  thanks for the comments. they are all fixed in the new commit https://github.com/awslabs/data-on-eks/pull/39/commits/d5130dd22af8f77131e56c201b33c85a0466ffd9

**Review by vara-bonthu — APPROVED** [review]
/LGTM

**Pull request merged**